doc/plotting.rst

```diff
@@ -375,9 +375,9 @@ Faceted plotting supports other arguments common to xray 2d plots.
     hasoutliers[0, 0, 0] = -100
     hasoutliers[-1, -1, -1] = 400
 
-    @savefig plot_facet_robust.png height=12in 
-    g = hasoutliers.plot.contourf('lon', 'lat', col='time', col_wrap=3,
-                                  robust=True, cmap='viridis')
+    @savefig plot_facet_robust.png height=12in
+    g = hasoutliers.plot.imshow('lon', 'lat', col='time', col_wrap=3,
+                                robust=True, cmap='viridis')
 
```

Comment:
Yes, that appears to be the case. I considered using `pcolormesh`, but that current displays the plots upside-down! (We might want to fix that?)